sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -845,3 +790,102 @@ def _latest_data_to_write(
     as_hashable = ((d[0].SerializeToString(), d) for d in data)
     sorted_data = sorted(as_hashable, key=lambda ah: (ah[0], ah[1][2]))
     return (v for _, v in OrderedDict((ah[0], ah[1]) for ah in sorted_data).items())
+
+
+class RetryableBotoError(Exception):
+    pass
+
+
+class LimitExceededException(RetryableBotoError):
+    pass
+
+
+class _DynamoTableManager:
```

Comment:
the online store class has become a [cluttered god class](https://linearb.io/blog/what-is-a-god-class), move the dynamo table creation/destruction to a helper class